eng/pipelines/templates/jobs/ci.yml

```diff
@@ -189,6 +189,17 @@ jobs:
            -SpellCheckPublicApiSurface
           pwsh: true
           failOnStderr: false
+      - script: >-
```

Comment:
Should we consider some kind of opt-in/out so that we can gradually transition everything that breaks, or are you thinking that we'd do it in one fell swoop?

Reply:
We're probably going to have to, if we even want to do this based on our internal Teams discussion.